gateway/src/apicast/policy/3scale_batcher/3scale_batcher.lua

```diff
@@ -160,13 +180,15 @@ end
 -- calls to backend.
 function _M:access(context)
   local backend = backend_client:new(context.service, http_ng_resty)
+self.backend = backend
   local usage = context.usage
   local service = context.service
   local service_id = service.id
+  self.service_id = service_id
```

Comment:
Why it is being assigned to self ?

Reply:
Sorry, this is a leftover from some test. Same with `self.backend`.

Reply:
Fixed.